__tests__/app.test.js

```diff
@@ -322,12 +322,12 @@ describe("GET /api/users", () => {
         expect(Array.isArray(data.body)).toBe(true);
       });
   });
-  it("responds with an array of users which contains a username, name and avatar URL", () => {
+  it.only("responds with an array of users which contains a username, name and avatar URL", () => {
```

Comment:
make sure to not push up .onlys

Reply:
I changed this a minute later you were just too fast for me!